server/application-server/src/main/java/de/tum/in/www1/hephaestus/gitprovider/pullrequestreview/PullRequestReviewInfoDTO.java

```diff
@@ -18,22 +19,24 @@ public record PullRequestReviewInfoDTO(
         UserInfoDTO author,
         PullRequestBaseInfoDTO pullRequest,
         @NonNull String htmlUrl,
+        @NonNull int score,
         OffsetDateTime submittedAt) {
     // We do not have createdAt and updatedAt for reviews
 
     public static PullRequestReviewInfoDTO fromPullRequestReview(PullRequestReview pullRequestReview) {
         return new PullRequestReviewInfoDTO(
-                pullRequestReview.getId(),
-                pullRequestReview.isDismissed(),
-                pullRequestReview.getState(),
-                pullRequestReview.getComments().size(),
-                UserInfoDTO.fromUser(pullRequestReview.getAuthor()),
-                PullRequestBaseInfoDTO.fromPullRequest(pullRequestReview.getPullRequest()),
-                pullRequestReview.getHtmlUrl(),
-                pullRequestReview.getSubmittedAt());
+            pullRequestReview.getId(),
+            pullRequestReview.isDismissed(),
+            pullRequestReview.getState(),
+            pullRequestReview.getComments().size(),
+            UserInfoDTO.fromUser(pullRequestReview.getAuthor()),
+            PullRequestBaseInfoDTO.fromPullRequest(pullRequestReview.getPullRequest()),
+            pullRequestReview.getHtmlUrl(),
+            (int) ScoringService.calculateReviewScore(List.of(pullRequestReview)),
```

Comment:
I guess that's why some people usually have a conversion service so we don't have to make it static. 🤔 
I don't know if it is problematic

Reply:
Although it's not super pretty, the `ScoringService` is pretty much stateless, so making it `static` isn't really an issue here.
I do see the general issue though.

So you would prefer creating a Converter over the `static`-approach here?